### DISCUSSION TITLE
neomuttlog.vim

### DISCUSSION THREAD

**Discussion by avidseeker** [neomutt/neomutt#4369] [Development]
Neomutt log includes a Vim modeline, but the file syntax is nowhere to find.

https://github.com/neomutt/neomutt/blob/8fea8f20e4c0d948974fd402adeba7691bc3dbf3/mutt/logging.c#L105

Where can I find it? Was is submitted to Vim then removed?

**Comment by gahr** [comment]
It's here: https://github.com/neomutt/neomutt.vim/blob/main/syntax/neomuttlog.vim
